cycledash/static/js/examine/components/ExaminePage.js

```diff
@@ -78,44 +78,57 @@ var ExaminePage = React.createClass({
         run = props.vcf;
     return (
       <div className="examine-page">
-        <StatsSummary hasLoaded={state.hasLoaded}
-                      stats={state.stats} />
-        <ExamineInformation run={run}/>
-        {props.comparableVcfs ?
-         <VCFValidation vcfs={props.comparableVcfs}
-                        selectedVcfId={state.selectedVcfId}
-                        handleComparisonVcfChange={this.handleComparisonVcfChange} /> :
-          null}
-        <QueryBox columns={state.columns}
-                  hasPendingRequest={state.hasPendingRequest}
-                  loadError={state.loadError}
-                  query={state.query}
-                  handleQueryChange={this.handleQueryChange} />
-        <Downloads query={state.query} run_id={run.id} />
-        <VCFTable ref="vcfTable"
-                  hasLoaded={state.hasLoaded}
-                  records={state.records}
-                  range={state.range}
-                  columns={state.columns}
-                  selectedRecord={state.selectedRecord}
-                  contigs={state.contigs}
-                  sortBys={state.sortBys}
-                  igvLink={state.igvLink}
-                  handleSortByChange={this.handleSortByChange}
-                  handleRequestPage={this.handleRequestPage}
-                  handleSelectRecord={this.handleSelectRecord}
-                  handleOpenViewer={this.handleOpenViewer}
-                  handleSetComment={this.handleSetComment}
-                  handleDeleteComment={this.handleDeleteComment} />
-        <PileupViewer vcfPath={run.uri}
-                      normalBamPath={run.normal_bam && run.normal_bam.uri}
-                      tumorBamPath={run.tumor_bam && run.tumor_bam.uri}
-                      igvHttpfsUrl={props.igvHttpfsUrl}
-                      selectedRecord={state.selectedRecord}
-                      isOpen={state.isViewerOpen}
-                      handlePreviousRecord={this.handlePreviousRecord}
-                      handleNextRecord={this.handleNextRecord}
-                      handleClose={this.handleCloseViewer} />
+        <div className="examine-header">
```

Comment:
I hear you on the nested. It's all for the Bootstrap grid, though. `.examine-header` is @extending Bootstrap's `.container`, class which consists of some `.row`'s that contain columns.

I'll give the refactoring a shot! But will probably push up these other changes first, since it may take me a while to figure out